src/pages/Login.jsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
-const Login = () => (
-    <div>
-        <a href="https://github.com/login/oauth/authorize?client_id=Iv1.ae8d32938d68d499">Login with GitHub</a>
-    </div>
-);
+const Login = ({ location: { pathname, search, hash } }) => {
+    localStorage.setItem('redirect-to', [pathname, search, hash].join(''));
+
+    return (
+        <div>
+            <a href="https://github.com/login/oauth/authorize?client_id=Iv1.ae8d32938d68d499">Login with GitHub</a>
```

Comment:
If we need to reuse the same client_id in other places I think we should move it in a constants.
Maybe the same could be for the _github basePath_

Reply:
The *right thing to do*™️ should be to turn the login URL to a prop and pass it from outside, from a constant or possibly a configuration file of some kind. But I think we can live with the static string for a while.